README.md

```diff
@@ -1 +1,76 @@
-# space-travelers
\ No newline at end of file
+# Space Travelers' Hub
+
+## Building a web application for a company that provides commercial and scientific space travel services. The application will allow users to book rockets and join selected space missions.
+
+## Project screenshots:
+
+#### 1. Rockets page
+![Rockets](./src/img/rockets.PNG)
+
+#### 2. Missions page
+![Missions](./src/img/missions.PNG)
+
+#### 3. My profile page
+![My profile](./src/img/profile.PNG)
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+- React
+- Redux
+- Netlify
+
+## Live Demo
+
+[Live Demo Link](https://space-travelers.netlify.app/)
```

Comment:
- [ ] The deployed project should also have the changes recommended by the previous reviewer. Kindly deploy the updated project.

<img src="https://user-images.githubusercontent.com/33323077/136513328-cf13e4a2-71d9-40d3-a1c0-ec46a92ab56b.png" height="250px" />